XmlFormatter/src/Update/VersionManagerFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿using XmlFormatterModel.Update;
+using XmlFormatterModel.Update.Strategies;
+
+namespace XmlFormatter.src.Update
+{
+    /// <summary>
+    /// Create a new version manager to use
+    /// </summary>
+    internal class VersionManagerFactory : IVersionManagerFactory
+    {
+        /// <summary>
+        /// The version manager to use
+        /// </summary>
+        private IVersionManager manager;
+
+        /// <summary>
+        /// Create a new instance of this class
+        /// </summary>
+        public VersionManagerFactory()
+        {
+            manager = null;
+        }
+
+        /// <inheritdoc/>
+        public IVersionManager GetVersionManager()
+        {
+            manager = manager ?? new VersionManager(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '??=' operator here.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)